### PR TITLE
Removed deprecated ntp set example

### PR DESCRIPTION
**Pull request by Zsoldier**
Remove set-NSXManager -ntpserver example since it does not appear to be part of function anymore.   Added help link to point to set-nsxmanagertimesettings function.

### CODE REVIEW

File: module/PowerNSX.psm1 [hunk 1]
```diff
@@ -5559,7 +5554,8 @@ function Set-NsxManager {
     Set-NsxManager -vcenterusername administrator@vsphere.local -vcenterpassword VMware1! -vcenterserver vcenter.corp.local
 
     Configures the vCenter registration of NSX Manager.
-
+    .LINK
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: Add space ?

**Comment by nmbradford** [comment]
Thanks @zsoldier.  Once you've signed the CLA I can merge this...

**Comment by Zsoldier** [comment]
Just waiting on my Virtustream legal team.  Thanks. :)

**Comment by Zsoldier** [comment]
New CLA was just signed.

**Comment by nmbradford** [comment]
Thanks @Zsoldier - did you click the 'here' in the comment from the clabot above?  The PR still has the cla-rejected label.  Might be worth raising another PR if it doesnt want to play?

**Comment by dcoghlan** [comment]
The changes to _init have already been implemented.
Closing this due to inactivity. Please reopen if still required.

**Pull request closed without merging**